pkgs/enchant/1.x.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, fetchurl, aspell, pkgconfig, glib, hunspell, hspell }:
```

Comment:
I do not really like keeping five year old packages. Ideally we would extract patches from the history https://github.com/php/php-src/commits/master/ext/enchant and bump our extension to enchant 2 but not sure it is worth the effort.

Reply:
I understand. In this case, `enchant2` is only [supported from PHP 8](https://www.php.net/manual/en/enchant.requirements.php), so I guess this PR is maybe the best option. No?

Reply:
Well, we could make it supported in older versions, like we are doing with OpenSSL 1.1. But I am not sure it is worth it. I doubt anyone even uses enchant from PHP.